### PR TITLE
Update Dev Mode instructions, add Dark mode preference

### PR DESCRIPTION
**Pull request by ghost**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Comment by vordimous** [comment]
This is my PR. I am consolidating my accounts and I should have closed before deleting the old account.

File: README.md [hunk 1]
```diff
@@ -13,7 +13,11 @@ To run the project on local, it requires [python](https://www.python.org/downloa
 ## Run project locally (Dev Mode)
 
 1. Clone repo
-2. Run `mkdocs serve` command in `<Lang folder>`. E.g. `cd en && mkdocs serve`
+1. Ensure Python and Pip are installed
```
Comment (gomathyk) [inline README.md:16]:
```suggestion
1. Ensure that Python and Pip are installed.
```

File: README.md [hunk 2]
```diff
@@ -13,7 +13,11 @@ To run the project on local, it requires [python](https://www.python.org/downloa
 ## Run project locally (Dev Mode)
 
 1. Clone repo
```
Comment (gomathyk) [inline README.md:15]: Give the link to the repo.

File: README.md [hunk 3]
```diff
@@ -13,7 +13,11 @@ To run the project on local, it requires [python](https://www.python.org/downloa
...
-2. Run `mkdocs serve` command in `<Lang folder>`. E.g. `cd en && mkdocs serve`
+1. Ensure Python and Pip are installed
+1. Install mkdocs cli tool `pip install mkdocs`
```
Comment (gomathyk) [inline README.md:17]:
```suggestion
1. Install the mkdocs cli tool using the command, `pip install mkdocs`.
```

File: README.md [hunk 4]
```diff
@@ -13,7 +13,11 @@ To run the project on local, it requires [python](https://www.python.org/downloa
...
+1. Install the material theme `pip install mkdocs-material`
```
Comment (gomathyk) [inline README.md:18]:
```suggestion
1. Install the material theme using the command, `pip install mkdocs-material`.
```

File: README.md [hunk 5]
```diff
@@ -13,7 +13,11 @@ To run the project on local, it requires [python](https://www.python.org/downloa
...
+1. Install the markdown extension `pip install markdown-include`
```
Comment (gomathyk) [inline README.md:19]:
```suggestion
1. Install the markdown extension using the command, `pip install markdown-include`.
```

File: README.md [hunk 6]
```diff
@@ -13,7 +13,11 @@ To run the project on local, it requires [python](https://www.python.org/downloa
...
+1. Run `mkdocs serve` command in `<Lang folder>`. E.g. `cd en && mkdocs serve`
```
Comment (gomathyk) [inline README.md:20]:
```suggestion
1. Run `mkdocs serve` in `<Lang folder>`, e.g., `cd en && mkdocs serve`
```

Comment (gomathyk) [inline README.md:20]: Give the folder path.

**Review by gomathyk — CHANGES_REQUESTED** [review]

**Comment by madurangasiriwardena** [comment]
@vordimous can you please check the above comments?

**Review by gomathyk — APPROVED** [review]

**Comment by vordimous** [comment]
@gomathyk @madurangasiriwardena I have addressed the comments above.

**Pull request merged**